index.js

```diff
@@ -59,25 +59,23 @@ export default class MarqueeLabel extends Component {
     } else {
       if (animation === null) {
         this.animatedTransformX.setValue(bgViewWidth);
-        this.setState({
+        await this.setState({
```

Comment:
According to [Component API - setState](https://shripadk.github.io/react/docs/component-api.html#setstate) and [source code](https://github.com/facebook/react/blob/master/packages/react/src/ReactBaseClasses.js#L53), `setState` does not return a Promise, so `await` is not doing anything here. 😉

But it does accept an optional callback.